web/packages/plugins/job/src/JobControl.tsx

```diff
@@ -123,10 +121,10 @@ export const JobControl = (props: {
         setJobLogs(result.data.message)
         setJobStatus(EJobStatus.STARTING)
       })
-      .catch((error: AxiosError<any>) => {
+      .catch((error: AxiosError<ErrorResponse>) => {
         console.error(error)
         NotificationManager.error(
-          (error.response && error.response.data?.message) || error.message,
+          error.response?.data.message || '',
```

Comment:
this might be enough, since it looks like a message is always defined in the API error classes by default

```suggestion
          error.response?.data.message
```